esmvaltool/config-developer.yml

```diff
@@ -26,6 +26,82 @@ CMIP5:
     SMHI: '[model]/[ensemble]/[exp]/[freq]'
   input_file: '[var]_[mip]_[model]_[exp]_[ensemble]_*'
   output_file: '[project]_[model]_[mip]_[exp]_[ensemble]_[field]_[var]_[start_year]-[end_year]'
+  institute:
```

Comment:
We should get this from somewhere automatically eventually, I guess. Great solution for now